Source/Scene/Model.js

```diff
@@ -4181,6 +4233,31 @@ define([
         }
     }
 
+    function modifyShaderForClippingPlanes(shader) {
+        shader = ShaderSource.replaceMain(shader, 'gltf_clip_main');
+        shader +=
+            'uniform int gltf_numClippingPlanes; \n' +
+            'uniform vec3 gltf_clipNormals[6]; \n' + // TODO Max doesn't have to be 6
+            'uniform vec3 gltf_clipPositions[6]; \n' +
+            'void main() \n' +
+            '{ \n' +
+            '    gltf_clip_main(); \n' +
```

Comment:
The clipping planes code should not be run if there are no clipping planes. Put an if in here that checks `gltf_numClippingPlanes > 0`